magda-migrator-authorization-db/sql/V1_4__default_dataset_resource_operations.sql

```diff
@@ -0,0 +1,16 @@
+INSERT INTO "public"."resources" ("id", "name", "uri", "description") 
```

Comment:
why split these up?

Reply:
v1.3 migration is fundamental things to describe the story --- structures & default values are likely never required to be changed even we decided to change our story.
v1.4 & up are an implementation of the story --- they are all setup initial data --- we may want to change it if we decide to change the story.
I was thinking it might be helpful to split it up if we want to change things while we work on the story. 😄 
Merge those files into one definitely will make DB migration complete quicker.
@AlexGilleran 
Do you think we should merge those file now or wait a bit later?

Reply:
If the data works, might as well merge them :)

Reply:
Merged them into one here (on PR for issue/2155):
https://github.com/magda-io/magda/pull/2184/commits/baf54711bcb872741003a4d1a4f5a8ac14aa8352

Reply:
OK I'll close this then :)